init

```diff
@@ -65,8 +65,14 @@ else
 fi
 
 DISTRO=$(operating_system)
-echo "ALL BUILD PACKAGES: $($DIR/environment/os/$DISTRO.sh list MEMGRAPH_BUILD_DEPS)"
-$DIR/environment/os/$DISTRO.sh check MEMGRAPH_BUILD_DEPS
+ARCHITECTURE=$(architecture)
+if [ "${ARCHITECTURE}" = "arm64" ]; then
+  echo "ALL BUILD PACKAGES: $($DIR/environment/os/$DISTRO-arm.sh list MEMGRAPH_BUILD_DEPS)"
+  $DIR/environment/os/$DISTRO.sh check MEMGRAPH_BUILD_DEPS
+else
+  echo "ALL BUILD PACKAGES: $($DIR/environment/os/$DISTRO.sh list MEMGRAPH_BUILD_DEPS)"
+  $DIR/environment/os/$DISTRO.sh check MEMGRAPH_BUILD_DEPS
+fi
```

Comment:
What about extracting the path to the correct script into a variable and using it? I think the arm branch has a bug in the checking, because it calls `$DISTRO.sh` instead of `$DISTRO-arm.sh`.

Reply:
You are right will do that